### ISSUE TITLE
New sorting option by score

### ISSUE THREAD

**Issue by zachvlat** [Heroic-Games-Launcher/HeroicGamesLauncher#3800]
### Problem description

The sorting option alphabetically and by status, are too little in my opinion. It's ok if you have a couple of games, but for large lists of games its not optimal to try to find a good game to play out of many.

### Feature description

What I suggest is a sorting option named "by score" from a third party api like igdb.com or something else.

**Comment by Aldijonas** [comment]
Yeah, I would love to sort by rating.  There are too many free games on epic, it is hard to determine which ones are worth checking out.
